bamboo/set-bamboo-env-variables.sh

```diff
@@ -42,6 +42,7 @@ if [[ $NGAP_ENV == 'SIT' ]]; then
 fi
 
 ## Get the current git SHA
+git config --global --add safe.directory $bamboo_working_directory/source/cumulus-dashboard
```

Comment:
Fix the ci build error:
--------------------------
fatal: detected dubious ownership in repository at '/data/bamboo/bamboo-agent-home/xml-data/build-dir/CUM-CDG239-SBE/cumulus-dashboard'
To add an exception for this directory, call:

        git config --global --add safe.directory /data/bamboo/bamboo-agent-home/xml-data/build-dir/CUM-CDG239-SBE/cumulus-dashboard
++ GIT_SHA=
+ source .bamboo_git_env
./bamboo/set-status.sh: line 4: .bamboo_git_env: No such file or directory